### DISCUSSION TITLE
PDF文档无法加载预览

### DISCUSSION THREAD

**Discussion by cvpas** [AlistGo/alist#925] [Q&A]
使用ipad网页访问alist上的阿里云盘，发现无法加载预览，excel、docx、mp3、mp4均可预览播放。
其他终端（iphone、MacBook）访问可以加载pdf预览。
![IMG_8846](https://user-images.githubusercontent.com/36500518/162602665-4d2ae545-a2f9-4724-8e9d-ccdd055ac54f.JPG)
![IMG_8847](https://user-images.githubusercontent.com/36500518/162602666-ed374cf2-82d3-41c1-852e-e0909f81f061.JPG)
![IMG_8848](https://user-images.githubusercontent.com/36500518/162602667-c8c0a4fc-5f9c-46a2-baa5-47a2e28f82c0.JPG)

**Comment by xhofe** [comment]
使用的是官方预览接口 目测是官方不支持。直接使用阿里云盘网页版在iPad上也打不开

**Comment by cvpas** [comment]
实测iPadOS上V3版本已经可以预览pdf。但是发现视频无法播放了😂
